HOWTO.md

```diff
@@ -0,0 +1,37 @@
+# How to use PEPC
+## Enabling PEPC
+### Chrome
+Use a version of Chrome that is version 121.0.6136 or higher. This might mean you need to download a 
+[canary](https://www.google.com/chrome/canary/), [dev](https://www.google.com/chrome/dev/) or 
+[beta](https://www.google.com/chrome/beta/) version of Chrome. You can check your current browser version
+by vising `chrome://version`.
+
+To enable the PEPC feature you need to start Chrome with the command line flag: `--enable-features=PermissionElement`.
+
+## Using PEPC
+You can visit https://permission.site/pepc for a quick example and test.
+
+To add the PEPC element to your page simply include it as you would any other HTML element. It does not have an end tag or contents.
+
+```
+<permission type="camera,microphone" ondismiss="pepcDismiss()" onresolve="pepcResolve()">
+```
+
+Current supported `type` attribute values (on Chrome) are: `"camera"`, `"microphone"` or both together `"camera, microphone"`.
+
+In order to make use of the permission you can use the permissions API to listen to permission status changes. This has the advantage
```

Comment:
Is there a "simple" version of this where you can start using the camera in the `onresolve` handler? Maybe the `Event` attribute could contain the permission state?

Reply:
That's not a pattern we want to encourage without having better support (e.g. an "ongranted" event). Sites using it will  not work when the permission is already granted so the permission API is inevitable.

Reply:
Gotcha. Let's mention maybe that for now this builds on the Permission API, but we are open to exploring usage patterns that do not involve the Permission API and are curious to hear feedback?

Reply:
Done